### ISSUE TITLE
Logging in v0.4.1 SentenceTransformer.encode does not seem to work as written on python 3.6

### ISSUE THREAD

**Issue by alexyang528** [UKPLab/sentence-transformers#802]
Console output with show_progress_bar = None:

```
  File "/usr/local/lib/python3.6/dist-packages/sentence_transformers/SentenceTransformer.py", line 155, in encode
    show_progress_bar = (logger.getLogger().getEffectiveLevel()==logger.INFO or logger.getLogger().getEffectiveLevel()==logger.DEBUG)
AttributeError: 'Logger' object has no attribute 'getLogger'
```

**Comment by nreimers** [comment]
When you update to the most recent version of sentence transformers, the bug should be fixed

**Comment by alexyang528** [comment]
I see, thanks Nils - I'm seeing the issue in 0.4.1, but will keep an eye out for when 0.4.2 is released.

**Comment by nreimers** [comment]
Should be fixed already with 0.4.1.2 (which you get when it is installed with pip)
